src/Helper/ContextHelper.php

```diff
@@ -66,6 +66,26 @@ public function isBoxedLayout(): bool
         return (bool) $this->getOption('boxed_layout');
     }
 
+    public function setLayout(string $layout): void
+    {
+        $this->setOption('layout', $layout);
+    }
+
+    public function getLayout(): string
```

Comment:
Thinking about it again: this could be private, right?

Reply:
Yes it could.
If you don't want to allow the user (of the bundle) get the real value behind `layout` option.

Reply:
Once you expose it, you have to make sure it works in the long run, which puts a higher maintenance burden on the maintainer => so using `final` and `private` wherever possible is the way to go. I don't see any value in leaking this information to the outside world.

Reply:
Ok I get it now. I'll make it private
Getter/Setter or only Getter ?

Reply:
The getter definitely and we keep on discussing about the setter 😁

Reply:
Updated with ccc4f46